tests/integrations/config/mod.rs

```diff
@@ -605,7 +605,7 @@ fn test_serde_custom_tikv_config() {
         enabled: false,
         wait_for_lock_timeout: ReadableDuration::millis(10),
         wake_up_delay_duration: ReadableDuration::millis(100),
-        pipelined: true,
```

Comment:
Please consider remove this change since false is the default value and  we want to test a different customized value.